### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by clopezr1**
Restructured the files
Added the ability to go to the sports page view
Currently:
Home can go to Sports
When in sports drop down appears for sports, the hamburger options change
sports selector drop down changes the sports view stuff.

### CODE REVIEW

**Comment by Bnice2max** [comment]
It works!
The explanation of the feature was unclear.  Also, pages weren't distinct so despite seeing the button it didn't feel responsive -> obviously will lead to a different page but a blank page with a  changing title & centered home button would have made this clear.

**Comment by Bnice2max** [comment]
image
Dropdown needs to be formatted.  see above^

**Comment by clopezr1** [comment]
> image
> Dropdown needs to be formatted. see above^

fixed

File: lib/screens/sport.dart [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  @override
+  Widget build(BuildContext context) {
+    Item selectedSport;
+
+    return StatefulBuilder(
+      builder: (context, StateSetter setState) => Scaffold(
+        appBar: AppBar(
+          centerTitle: false,
+          title: buildDropdownButton(selectedSport, setState),
+          backgroundColor: _curSport[1],
```
Comment (Bnice2max) [inline lib/screens/sport.dart]: no. bad. _curSport[#] should have a  _curSport.color or some other naming convention which if you create curSport as: 
  Item _curSport = colorList[0]; 
_curSport.name
_curSport = value; //no need to deep copy

Comment (Bnice2max) [reply lib/screens/sport.dart]: or use: Item get _curSport => colorList[0];

**Pull request merged**